services/QuillLMS/spec/serializers/progress_reports/standards/student_serializer_spec.rb

```diff
@@ -58,5 +59,15 @@
     it 'includes properly rounded scores' do
       expect(parsed_student['average_score']).to eq(0.75)
     end
+
+    it 'should not raise error when average_score is nil' do
+      mock_object = double.tap { |mock| allow(mock).to receive(:average_score).and_return(nil) }
+      serializer = described_class.new(mock_object)
+      expect do
```

Comment:
A couple of stylistic suggestions:
1) Since we're not using `mock_object` elsewhere, you could just pass the double in directly `serializer = described_class.new(double(average_score: nil))` and remove the mock_object assignment altogether.

2) Make the expectation a one-liner since it's a one-line block:  `expect { serializer.average_score }.not_to raise_error`